setup.py

```diff
@@ -13,15 +13,11 @@
 ##############################################################################
 """Setup for zc.table package
 """
-import os
 from setuptools import setup, find_packages
 
-def read(*rnames):
-    return open(os.path.join(os.path.dirname(__file__), *rnames)).read()
-
 setup(
     name="zc.table",
-    version='0.9.1dev',
+    version='0.10.0.dev0',
     url="http://pypi.python.org/pypi/zc.table/",
```

Comment:
This URL is now a redirect. It's most common to have this be the GitHub homepage.

Reply:
I fixed the url